coral/src/app/layout/header/HeaderMenuLink.tsx

```diff
@@ -1,33 +1,23 @@
 import { Icon, Tooltip } from "@aivenio/aquarium";
 import data from "@aivenio/aquarium/dist/src/icons/console";
 import { useState } from "react";
+import { Link } from "react-router-dom";
 
 type HeaderMenuLinkProps = {
   icon: typeof data;
   linkText: string;
   href: string;
   rel?: string;
+  externalLink?: boolean;
```

Comment:
I'm a little confused because I see the merit in adding this option, but I don't see it used anywhere. I expected it in `HeaderNavigation` :o Am I missing something?

Reply:
Nope, I missed something -> adding the property where I thought we need it (the link to the docs, which is not internal). And I missed that with react routers `Link`, if you add a `to` that has a full url, that will work just fine 🤦‍♀️ 🤦‍♀️ 🤦‍♀️ 🤦‍♀️ 🤦‍♀️ 🤦‍♀️ 🤦‍♀️ 🤦‍♀️   So replacing the `a` with `Link` does solve the wrong link for `/approvals`. 

https://github.com/Aiven-Open/klaw/pull/1763/commits/5cb61924bafe934d45b1a6158e21149ff51c32e8

Reply:
Nice ^^